test/Router/IntegrationTest.php

```diff
@@ -103,6 +106,7 @@ private function createApplicationWithGetPost($adapter, $getName = null, $postNa
         $router = new $adapter();
         $app = $this->createApplicationFromRouter($router);
         $app->pipe(new RouteMiddleware($router, $this->response));
```

Comment:
Here and in many other usages - the second parameter should be remove, `RouteMiddleware::__construct` has now one parameter.

Reply:
Another nice catch! Fixing!